<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>399</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>399</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>399</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>600</width>
    <height>399</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>399</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>CADASTRO CLI-V1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-6</y>
      <width>700</width>
      <height>621</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 127);
border-radius: 7px;
border-style: outset;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="lb_codigo">
     <property name="geometry">
      <rect>
       <x>26</x>
       <y>26</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>10</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);
background-color: rgb(152, 152, 152);

</string>
     </property>
     <property name="text">
      <string>Código</string>
     </property>
    </widget>
    <widget class="QLabel" name="tx_nome">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>10</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);
background-color: rgb(152, 152, 152);

</string>
     </property>
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_telefone">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>10</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);
background-color: rgb(152, 152, 152);

</string>
     </property>
     <property name="text">
      <string>Telefone:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset; 
border-width: 2px; </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset; 
border-width: 2px; </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>                 (xx)xxxxx-xxxx</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>342</x>
       <y>180</y>
       <width>321</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset; 
border-width: 2px; </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>    gmail.com - hotmail.com - yahoo.com</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Sans Serif</family>
       <pointsize>17</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border-style: outset; 
border-width: 2px; </string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string> *****</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>641</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset; 
border-width: 2px; </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Ex: Rua Presidente lima vargas </string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_telefone_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>10</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);
background-color: rgb(152, 152, 152);
</string>
     </property>
     <property name="text">
      <string>Endereço</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_telefone_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>160</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>10</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);

background-color: rgb(152, 152, 152);</string>
     </property>
     <property name="text">
      <string>E-mail:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>-47</y>
       <width>400</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>8</y>
       <width>695</width>
       <height>375</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font/>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(152, 152, 152);
border-radius: 20px;

border-width: 2px; 
border-radius: 5px;
border-style: outset; 
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>175</x>
        <y>-47</y>
        <width>400</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_novo">
      <property name="geometry">
       <rect>
        <x>342</x>
        <y>24</y>
        <width>249</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(106, 106, 106);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-width: 2px; 
border-radius: 10px;
border-style: outset; 
border-color: rgb(0, 0, 0);
}

QPushButton:pressed{
background-color: rgb(106, 106, 106);
color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
border-width: 4px; 
border-radius: 10px;
border-style: outset; 
border-color: rgb(255, 255, 255);



}
 </string>
      </property>
      <property name="text">
       <string>Adicionar novo Cliente </string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>imagens Projeto/novo 01.png</normaloff>imagens Projeto/novo 01.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>290</y>
        <width>299</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 15px;
border-style: outset; 
border-width: 2px; </string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>xxx.xxx.xxx-xx</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_buscar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>114</x>
        <y>30</y>
        <width>122</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>12</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(106, 106, 106);
color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-width: 2px; 
border-radius: 10px;
border-style: outset; 
border-color: rgb(0, 0, 0);
}

QPushButton:pressed{
background-color: rgb(106, 106, 106);
color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
border-width: 4px; 
border-radius: 10px;
border-style: outset; 
border-color: rgb(255, 255, 255);



}
 </string>
      </property>
      <property name="text">
       <string>Clientes</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>imagens Projeto/lupa.png</normaloff>imagens Projeto/lupa.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="bt_sair">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>300</y>
        <width>115</width>
        <height>61</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background-color: rgb(152, 152, 152);

color: rgb(255, 255, 255);
font: 87 12pt &quot;Arial Black&quot;;
border-width: 0px; 
border-radius: 0px;
border-style: outset; 
border-color: rgb(0, 0, 0);
}

QPushButton:pressed{
background-color: rgb(106, 106, 106);
color: rgb(255, 255, 255);
font: 87 10pt &quot;Arial Black&quot;;
border-width: 4px; 
border-radius: 10px;
border-style: outset; 
border-color: rgb(255, 255, 255);



}
 </string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>imagens Projeto/logout.png</normaloff>imagens Projeto/logout.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="lb_telefone_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>278</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>10</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 87 10pt &quot;Arial Black&quot;;
color: rgb(0, 0, 0);
background-color: rgb(152, 152, 152);
</string>
     </property>
     <property name="text">
      <string>CPF:</string>
     </property>
    </widget>
    <zorder>frame_2</zorder>
    <zorder>label</zorder>
    <zorder>lb_codigo</zorder>
    <zorder>tx_nome</zorder>
    <zorder>lb_telefone</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>lineEdit_4</zorder>
    <zorder>lineEdit</zorder>
    <zorder>lineEdit_9</zorder>
    <zorder>lb_telefone_3</zorder>
    <zorder>lb_telefone_2</zorder>
    <zorder>lb_telefone_5</zorder>
    <zorder>lineEdit_3</zorder>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>-10</y>
      <width>685</width>
      <height>581</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>frame_1</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGGGGG">
    <property name="title">
     <string>Opções</string>
    </property>
   </widget>
   <addaction name="menuGGGGG"/>
  </widget>
 </widget>
 <tabstops>
  <tabstop>bt_buscar</tabstop>
  <tabstop>bt_novo</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_5</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
